MIGRATING_TO_V8.0.0.MD

```diff
@@ -0,0 +1,133 @@
+- You may then enable double encoding for an rql builder `rqlBuilder({ doubleEncodeValues: true }).eq('diagnosis', 'Hypertension - STAGE 1')`
+- You may manually encode values for an rql builder `rqlBuilder().eq('diagnosis', encodeURIComponent('Hypertension - STAGE 1'))`
+
+For more information regarding manual double encoding please refer to the [RQL documentation](https://docs.extrahorizon.com/extrahorizon/additional-resources/resource-query-language-rql#double-encoding-of-special-characters).
+
+## Disabling custom key normalization
+
+### Changes required in requests
+
+For all affected properties with the request mentioned, check if you use any property names that are in `camelCase` and change them to `snake_case`.
+
+Affected properties in requests had their keys transformed from camelCase to snake_case before sending it to the service.
+An example would be all properties in the `customFields` property of the profile service. Updating this field with a value of
+`{ 'helloWorld': 'test' }` would result in the value being stored as `{ 'hello_world': 'test' }`. 
+
+### Changes required in  responses
+
+For all affected properties with the response mentioned, check if you use any property names that are in `snake_case` and start addressing them as `snake_case` instead of `camelCase`.
+This includes all properties that were normalized in the requests.
+
+For all affected properties with the response mentioned, check if you use any property as if it is a date. If you do, transform it to a Date (`new Date(property)`) before using it. 
+
+Affected properties in responses had their keys transformed from snake_case to camelCase before returning the data.
+An example would be all properties in the `data` property of the configurations service. Returning a field with a value of
+`{ 'hello_world': 'test' }` would result in the value being retrieved as `{ 'helloWorld': 'test' }`.
+
+Timestamps in responses of affected properties were sometimes converted to an actual javascript data object. 
+They will now always come as what was provided in the request. If a property was provided as Date, it will be returned as a string.
+
+### Reverting to request normalization of custom keys for a single operation
+You may enable normalization of custom keys by setting the option `sdk.service.operation({ normalizeCustomData: true })`
+- Before 8.0.0: `sdk.profiles.update(id, data, options)`
+- After 8.0.0: `sdk.profiles.update(id, data, {...options, normalizeCustomData: true ))`
+
+### Reverting to request normalization of custom keys for all operation a client
+The normalization of custom keys can be reverted to behavior before 8.0.0 using the snippet `sdk = createClient({ ...options, normalizeCustomData: true });`.
+- You may then disable normalization of custom keys for a single operation by  `sdk.service.operation({ normalizeCustomData: false })`
+
+### Affected Properties
+
```

Comment:
(minor) A little into sentence like `These are the affected properties, listed by the relevant SDK method:`?